src/Datadog.Trace.ClrProfiler.Native/cor_profiler.cpp

```diff
@@ -40,17 +40,32 @@ CorProfiler::Initialize(IUnknown* cor_profiler_info_unknown) {
     return E_FAIL;
   }
 
-  // check if there is a whitelist of process names
-  const auto allowed_process_names =
-      GetEnvironmentValues(environment::process_names);
+  // check if there is a process name inclusion list
+  const auto include_process_names =
+      GetEnvironmentValues(environment::include_process_names);
 
-  if (!allowed_process_names.empty()) {
+  if (!include_process_names.empty()) {
     const auto process_name = GetCurrentProcessName();
```

Comment:
Would it save us much to consolidate the calls to `GetCurrentProcessName`?

Reply:
Good catch. Refactored to only call once.